pkg/solana/chainwriter/transform_registry_test.go

```diff
@@ -0,0 +1,60 @@
+package chainwriter_test
+
+import (
+	"testing"
+
+	"github.com/gagliardetto/solana-go"
+	"github.com/smartcontractkit/chainlink-ccip/pkg/types/ccipocr3"
+	"github.com/smartcontractkit/chainlink-common/pkg/utils/tests"
+	"github.com/stretchr/testify/require"
+
+	"github.com/smartcontractkit/chainlink-solana/pkg/solana/chainwriter"
+)
+
+func Test_CCIPCommitTransform(t *testing.T) {
```

Comment:
Should we move the [CCIP execute ArgsTransform test](https://github.com/smartcontractkit/chainlink-solana/blob/e15b1333a14ad5d59aa6cc5407570d373cb3604e/pkg/solana/chainwriter/chain_writer_test.go#L861-L923) here or add this test to `chain_writer_test`?

Reply:
I should maybe do both tbh. The test in `chain_writer_test.go` tests this in the context of the entire CW with the TXM. My test here is just a unit test on the specific function. I should probably do both for both functions

Reply:
Yeah that's probably the safest option